### DISCUSSION TITLE
How to change path of default directories -  Windows 10

### DISCUSSION THREAD

**Discussion by bnpwalter** [freeplane/freeplane#810] [Help]
Installed freeplan 1.10.5 in folder "E:\Program Files\Freeplane"   My C-drive is an SSD drive capacity of 111 GB.  My E-Drive is SATA capacity 931 GB.  

When I save a map the path is "E:\Program Files\Freeplane\Doc"  
When I File/Open Mind Map the default path is "C:\Users\bnpwa\Documents"

How do I change the location where my maps are stored? 
How do I change the default location to where I store my maps?

This is a follow-up on my issue with default folder locations. I opened the freeplanUserGuide and some of my child nodes are not being found. I get the message: "FileNotFoundException E:_FreePlane Files|Images|mouse\mouse_RB.png (The system cannot find the path specified: Images/mouse/mouse_RB.png. This was after I saved a copy of the file to a location called "E:_Freeplane Files" and restarted FP.

**Comment by bnpwalter** [comment]
![FileNotFoundException](https://user-images.githubusercontent.com/50636483/200480467-c4b87ea2-1a8e-422c-b4ea-7e9724a50fe5.png)

**Comment by bnpwalter** [comment]
As you may tell I'm pretty confused on how to change the default directories in FP.  I would like FP to save .mm files and saves into my E-drive not the C-drive.  Going into the Windows Registry Editor I don't see FreePlane under HKEY_LOCAL_MACHINE\SOFTWARE.  I change the Save directory under Tools\Preferences\Environment to 

**E:\_Seva\_Satsang Work\_0 FreePlane Files**
![FP_Preferences_Save](https://user-images.githubusercontent.com/50636483/200723797-7c0d99bc-2d94-4381-ba67-56854e4f9f79.png)

On selecting the **Save** button instead I'm directed to **C:\Users\bnpwa\AppData\Roaming\Freeplane\1.10x**
![FP_Preferences Folder](https://user-images.githubusercontent.com/50636483/200723817-63f56b09-bc3b-4b67-bb62-21e29a42cb7e.png)

Freeplane launches from "E:\Program Files\Freeplane" directory.   It's unclear why FP is referencing my C-drive at all.  
![FP_Properties_Target location](https://user-images.githubusercontent.com/50636483/200724539-e2738a93-1e6e-4661-9dd3-1884b7026d5a.png)

Any guidance would be appreciated.  Thanks.

**Comment by engdarkalchemist** [comment]
Hi [bnpwalter](https://github.com/bnpwalter).

Did you find any solution? I have a similar issue.
